### PR TITLE
Update to Async `ElementHandle`

### PR DESCRIPTION
**Pull request by inancgumus**
## What?

Update relevant `ElementHandle` methods from sync to async.

## Checklist

- [x] I have used a meaningful title for the PR.
- [x] I have described the changes I've made in the "What?" section above.
- [x] I have performed a self-review of my changes.
- [x] I have run the `npm start` command locally and verified that the changes look good.

- [x] I have made my changes in the `docs/sources/next` folder of the documentation.
- [ ] I have reflected my changes in the `docs/sources/v{most_recent_release}` folder of the documentation.
- [ ] I have reflected my changes in the relevant folders of the two previous k6 versions of the documentation (if still applicable to previous versions).

## Related PR(s)/Issue(s)

Updates: grafana/xk6-browser#1296

### CODE REVIEW

File: docs/sources/next/javascript-api/k6-experimental/browser/elementhandle/waitforselector.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
+---
+title: 'waitForSelector(selector[, options])'
+description: 'Browser module: elementHandle.waitForSelector(selector[, options]) method'
+---
+
+# waitForSelector(selector[, options])
```
Comment (ankur22) [inline docs/sources/next/javascript-api/k6-experimental/browser/elementhandle/waitforselector.md:6]: Does this need to point to `locator.waitFor`?

**Review by ankur22 — APPROVED** [review]
Nice work! I may have missed somethings though since there were a lot of changes.

I was able to catch something which need to be reviewed.

**Comment by inancgumus** [comment]
> Nice work! I may have missed somethings though since there were a lot of changes.

Thanks! I've fixed what you could find [here](https://github.com/grafana/k6-docs/pull/1598/commits/e994ed8c92d3676b7d50acc0e99d416deb616a9b).

**Comment by inancgumus** [comment]
@ankur22 

> Nice work! I may have missed somethings though since there were a lot of changes.

Thanks! I've fixed what you could find [here](https://github.com/grafana/k6-docs/pull/1598/commits/e994ed8c92d3676b7d50acc0e99d416deb616a9b).

**Review by ankur22 — APPROVED** [review]
LGTM

**Review by allansson — APPROVED** [review]
LGTM 👍

**Pull request merged**